extensions/wikia/AdEngine/js/provider/factoryWikiaGpt.js

```diff
@@ -77,8 +63,12 @@ define('ext.wikia.adEngine.provider.factory.wikiaGpt', [
 				lookups.extendSlotTargeting(slotName, slotTargeting);
 			}
 
-			gptHelper.pushAd(slotName, slotPath, slotTargeting, doSuccess, doHop);
-			flushIfNeeded();
+			if (extra.sraEnabled) {
```

Comment:
I'm still thinking if we need this condition here. We have two ways of using `GptSraHelper`:
* use it instead of `GptHelper` in a provider (example: `turtle.js`),
* use it conditionally in a provider - how we make it here.

And I wonder if we could choose one of those way and stick to it. So it's more consistent. My suggestion would be using `GptSraHelper` instead of `GptHelper` - then we could remove also `sraEnabled` flag, I guess. But I have the feeling I'm missing something here and we have to leave the condition here because of other partners.